src/inflators/video-loader.ts

```diff
@@ -1,10 +1,10 @@
 import { HubsWorld } from "../app";
-import { ProjectionMode } from "../utils/projection-mode";
+import { MediaVideoLoaderData } from "../bit-components";
 import { inflateMediaLoader } from "./media-loader";
 
 export interface VideoLoaderParams {
   src: string;
-  projection: ProjectionMode;
+  projection?: string;
```

Comment:
Not blocking issue but just a question: Why only `projection` is optional in `VideoLoaderParams`?

Reply:
I don't think it is probably a copy&paste issue, updated